examples/nextjs/src/components/shared/contentlets.js

```diff
@@ -11,35 +13,39 @@ function Contentlets({ contentlets }) {
     return (
         <ul className="flex flex-col gap-7">
             {contentlets.map((contentlet) => (
-                <li key={contentlet.identifier} className="flex gap-7 min-h-16">
-                    <a
-                        className="relative min-w-32"
-                        href={contentlet.urlMap || contentlet.url}
-                    >
-                        <Image
-                            src={`${process.env.NEXT_PUBLIC_DOTCMS_HOST}${
-                                contentlet.image
-                            }?language_id=${contentlet.languageId || 1}`}
-                            alt={contentlet.urlTitle}
-                            fill={true}
-                            className="object-cover"
-                        />
-                    </a>
-                    <div className="flex flex-col gap-1">
+                <Contentlet contentlet={contentlet} key={contentlet.identifier}>
+                    <li className="flex gap-7 min-h-16">
                         <a
-                            className="text-sm text-zinc-900 font-bold"
+                            className="relative min-w-32"
                             href={contentlet.urlMap || contentlet.url}
                         >
-                            {contentlet.title}
+                            <Image
```

Comment:
@zJaaal In the same case as in the Angular example, we could add a function to resolve the src

The doc => https://nextjs.org/docs/pages/api-reference/next-config-js/images

Reply:
We actually have an issue where we will probably have to expose a function that takes care of the images, because we are having errors with inconsistence data from the backend. So we can probably use this approach after we make that fix 

cc: @fmontes